src/js/services/conversation-service.js

```diff
@@ -89,7 +89,7 @@ export function sendMessage(text) {
 
         const {user} = store.getState();
 
-        return core().conversations.sendMessage(getUserId(), message).then((response) => {
+        return core().appUsers.sendMessage(getUserId(), message).then((response) => {
             if (!user.conversationStarted) {
```

Comment:
But that logic is bogus now isn't it? When you create a conversation, it doesn't include the messages in the response anymore, so we need to update the local message to match the one on the server (which is what I think this `else` is doing)